graph/resolvers/asset.js

```diff
@@ -1,4 +1,11 @@
 const Asset = {
+  lastComment({id}, _, {loaders: {Comments}}) {
+    return Comments.getByQuery({
+      asset_id: id,
+      limit: 1,
+      parent_id: null
+    }).then(data => data[0]);
```

Comment:
1. I checked that and the graphql code is handling this well, it basically returns undefined and graphql translate that to `null`. Should I still do a check? :-D